internal/models/policy/recipe/custom/tmc_block_resources.go

```diff
@@ -0,0 +1,51 @@
+/*
+Copyright © 2022 VMware, Inc. All Rights Reserved.
+SPDX-License-Identifier: MPL-2.0
+Code generated by go-swagger; DO NOT EDIT.
+*/
+
+package policyrecipecustommodel
+
+import (
+	"github.com/go-openapi/swag"
+
+	policyrecipecustomcommonmodel "github.com/vmware/terraform-provider-tanzu-mission-control/internal/models/policy/recipe/custom/common"
+)
+
+// VmwareTanzuManageV1alpha1CommonPolicySpecCustomV1TMCBlockResources tmc-block-resources recipe schema.
+//
+// The input schema for tmc-block-resources recipe.
+//
+// swagger:model VmwareTanzuManageV1alpha1CommonPolicySpecCustomV1TMCBlockResources
+type VmwareTanzuManageV1alpha1CommonPolicySpecCustomV1TMCBlockResources struct {
```

Comment:
nodeport and block resources are same right ? what is the difference

Reply:
Yeah since these are dependent on the gatekeeper CR spec which may be updated, thus we have kept the high level structs for the recipes different although they might appear similar for now.

Anything that is under one recipe kind and is repeated while generating models can be reused same applies for any non-primitive object model across different policies which is embedded inside the top level recipe structs.

Reply:
I could not get what is the take away here. 
But I would suggest to avoid duplicating the structs if the same struct can we used again